js/core/clusters/clusters.js

```diff
@@ -140,7 +162,7 @@ define([
         bindClusterButtons();
 
         // select first cluster once clusters informations have been retrieved
-        $(document).trigger('selectCluster', { clusterId: 0, $cluster: $('.cluster').first() });
+        return $(document).trigger('selectCluster', { clusterId: 0, $cluster: $('.cluster').first() });
```

Comment:
Is this change related to the other changes of this commit? If not, this should be a separated commit.